source/PluginDev/Assets/GooglePlayGames/BasicApi/PlayGamesClientConfiguration.cs

```diff
@@ -180,7 +204,7 @@ public Builder EnableSavedGames()
                 mEnableSaveGames = true;
                 return this;
             }
-
+            
```

Comment:
done, thanks

On Wed, Jul 27, 2016 at 5:34 PM, Clayton Wilkinson <notifications@github.com
> wrote:

> In
> source/PluginDev/Assets/GooglePlayGames/BasicApi/PlayGamesClientConfiguration.cs
> <https://github.com/playgameservices/play-games-plugin-for-unity/pull/1296#discussion_r72450609>
> :
>
> > @@ -180,7 +204,7 @@ public Builder EnableSavedGames()
> >                  mEnableSaveGames = true;
> >                  return this;
> >              }
> > -
> > +
>
> minor issue with spaces on blank lines in a couple places. Please consider
> cleaning them up, but it won't hold up the merge.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/playgameservices/play-games-plugin-for-unity/pull/1296/files/98ba1f173583a90c3c7f3812b183e6800059c101#r72450609>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMcrIZUPNcLWgp1MJACtqTa9zk-gqZ3oks5qZ2xngaJpZM4JU7vu>
> .
>



-- 
Regards,
Paul